statchatui/src/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./SignIn.css";
```

Comment:
It looks like this file doesn't exist. Is that right, or was it not committed?

If it doesn't exist, let's remove this import and all the references to it.

Reply:
the file exists for me locally but it is empty, i will see why this is not showing on github

Reply:
the file is on github but it is blank